src/Workspaces/Core/Portable/Options/IOptionWithOptionGroup.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Options
+{
+    /// <summary>
+    /// Group/sub-feature associated with an <see cref="IOption"/>.
+    /// </summary>
+    internal interface IOptionWithOptionGroup : IOption
```

Comment:
naming.  IOption2, IOptionWithGroup.  

we should consider if we can just change IOption, but htat's a larger discussion since it is public.  we need to see if TS/F#/Xaml actually *implement* IOption themselves.

Reply:
> IOption2, IOptionWithGroup.  

I originally used IOptionWithGroup :), can't recall why I switched to this name, but can go back to that name.

> we should consider if we can just change IOption

We probably need a new interface to avoid compat issue for implementors. Regardless, I will file a tracking issue for designing the public API surface.

---
In reply to: [219246900](https://github.com/dotnet/roslyn/pull/30043#discussion_r219246900) [](ancestors = 219246900)

Reply:
[https://github.com/dotnet/roslyn/issues/30085](https://github.com/dotnet/roslyn/issues/30085)

---
In reply to: [219314838](https://github.com/dotnet/roslyn/pull/30043#discussion_r219314838) [](ancestors = 219314838,219246900)